src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2534,9 +2534,11 @@ namespace NServiceBus.Transports
     }
     public class PushContext
     {
-        public PushContext(NServiceBus.Transports.IncomingMessage message, NServiceBus.Extensibility.ContextBag context) { }
+        public PushContext(string messageId, System.Collections.Generic.Dictionary<string, string> headers, System.IO.Stream bodyStream, NServiceBus.Extensibility.ContextBag context) { }
```

Comment:
Since the transport is the owner of the original headers should we make them readonly?